src/styles/global.tsx

```diff
@@ -11,7 +11,7 @@ const globalStyle = css`
   body {
     color: ${colors.grayScale90};
     font-weight: normal;
-    font-family: Pretendard, serif;
+    font-family: 'Pretendard', 'serif';
```

Comment:
```suggestion
    font-family: 'Pretendard';
```
아하! 감사합니다!

serif 제공은 안해도 될 것 같은데 제거하는 거 어떠신가요??

Reply:
stylelint 룰에 default style이 설정되어야해서 추가해놓았는데 룰을 제거하는게 나을까용 ?